### PR TITLE
fix: `anime` file structure

### PR DESCRIPTION
**Pull request by tokitouq**
**NVM** this

### CODE REVIEW

File: backend/django_core/templates/anime/_layout.html [hunk 1]
```diff
@@ -142,11 +142,11 @@
...
                 <div id="page" class="flex-1 overflow-x-hidden flex flex-col" style="scrollbar-gutter: auto;">
                     {% if request.path == "/anime/" %}
-                        {% include 'anime/_partial_home.html'%}
+                        {% include 'anime/_partials/home.html'%}
                     {% elif request.path == "/anime/explore/" %}
-                        {% include 'anime/_partial_explore.html'%}
+                        {% include 'anime/_partials/explore.html'%}
                     {% elif request.resolver_match.view_name == 'anime_info_view' %}
-                        {% include 'anime/_partial_info.html'%}
+                        {% include 'anime/_partials/info.html'%}
```
Comment (baseplate-admin) [inline backend/django_core/templates/anime/_layout.html:149]: I dont think this is the better way of doing things. Check the register/login page for how i implemented things

File: backend/django_core/apps/pages/views/anime.py [hunk 2]
```diff
@@ -26,7 +26,7 @@ async def anime_home_view(request: "HtmxHttpRequest") -> HttpResponse:
 
 async def anime_explore_view(request: "HtmxHttpRequest") -> HttpResponse:
     if request.htmx:
-        return render(request, "anime/_partial_explore.html")
+        return render(request, "anime/_partials/explore.html")
```
Comment (baseplate-admin) [inline backend/django_core/apps/pages/views/anime.py:29]: same

File: backend/django_core/templates/anime/_layout.html (hunk 1 above)
Comment (tokitouq) [reply backend/django_core/templates/anime/_layout.html]: > Check the register/login page for how i implemented things

yea I tried that but reverted, uhh I forgot the reason
you mean **extending** right?

Comment (baseplate-admin) [reply backend/django_core/templates/anime/_layout.html]: NOOO . Check the register page lgoic

Comment (tokitouq) [reply backend/django_core/templates/anime/_layout.html]: Yeaa got it :sparkles:

Comment (baseplate-admin) [reply backend/django_core/templates/anime/_layout.html]: https://github.com/baseplate-admin/CoreProject/blob/93662b1b3b163591964af248b653efff2e16756b/backend/django_core/templates/user/register/index.html#L18-L23

**Pull request merged**